src/main/java/Config/Config.java

```diff
@@ -0,0 +1,69 @@
+package Config;
+
+import Config.Routes.Route;
+import Config.Routes.RouteFactory;
+import Config.Routes.SimpleRouteFactory;
+import Controller.Handler.GetEchoHandle;
+import Controller.Handler.GetHandle;
+import Controller.Handler.Handler;
+import Controller.Handler.PostEchoHandle;
+
+import java.io.IOException;
+import java.net.Socket;
+import java.net.SocketException;
+import java.util.ArrayList;
+
+import static Config.Method.GET;
+import static Config.Method.POST;
+
+import static Validator.Validator.validArgsLength;
+import static Validator.Validator.validFlag;
+
+public class Config {
+  public static ArrayList<Route> routes = new ArrayList<>();
+  public static ArrayList<Handler> handlers = new ArrayList<>();
+
+  public static void createRoutes() {
+    SimpleRouteFactory factory = new SimpleRouteFactory();
+    RouteFactory routeFactory = new RouteFactory(factory);
+    routes.add(routeFactory.createRoute(GET, "/", new GetHandle()));
+    routes.add(routeFactory.createRoute(GET, "/echo", new GetEchoHandle()));
+    routes.add(routeFactory.createRoute(POST, "/echo", new PostEchoHandle()));
```

Comment:
I really dig this structure 👍 

I notice that `routes` is an `ArrayList` - have you considered making them a [collection object](http://black-saturn.com/the-ruby-collector.html)?